source/appModules/eclipse.py

```diff
@@ -17,14 +22,83 @@ def event_valueChange(self):
 		# Therefore, just drop this event.
 		pass
 
+class AutocompletionListView(IAccessible):
+
+	def _get_name(self):
+		return AUTOCOMPLETION_LIST_NAME
+
+class AutocompletionListItem(IAccessible):
+
+	def event_selection(self):
+		# Fixme: I don't know if it is the correct way to do this.
+
+		# I think that you can feel more confortable if the text editor gain the focus after the item becomes hidden.
+		if self.appModule.textEditor == None:

```

Comment:
nit: Don't use "== None". Either use "is None" (faster) or just the "not" operator if there's no chance it could be False or 0. I think the latter is better in this case:

`if not self.appModule.textEditor:`